sycl/source/detail/xpti_registry.hpp

```diff
@@ -65,6 +69,9 @@ class XPTIRegistry {
       // SYCL buffer events
       GBufferStreamID = xptiRegisterStream(SYCL_BUFFER_STREAM_NAME);
       this->initializeStream(SYCL_BUFFER_STREAM_NAME, 0, 1, "0.1");
+      // SYCL image events
```

Comment:
@steffenlarsen It is okay to have a new stream for experimental data, but once it is validated, please ensure that the data is emitted in the default "sycl" stream. Everything that is a part of SYCL spec needs to be in the "sycl" stream.

Reply:
@steffenlarsen Having separate streams with common tracepoint types is a reasonable way to do this and limit the explosion of trace point types. When done with validating and settling on the 2020 image implementation, consider moving from "sycl.experimental.image" to "sycl.image"